docs-app/ember-cli-build.js

```diff
@@ -14,10 +18,50 @@ module.exports = function (defaults) {
     },
   });
 
-  return require('@embroider/compat').compatBuild(app, Webpack, {
+  const options = {
     packagerOptions: {
+      cssLoaderOptions: {
+        modules: {
+          localIdentName: isProduction()
+            ? '[sha512:hash:base64:5]'
+            : '[path][name]__[local]',
+          // Enable local mode only for CSS files from the host app
+          mode: (resourcePath) => {
+            // The host app and active child addons are moved into a common
+            // stable temp dir (`options.workspaceDir`), before the `css-loader`
+            // processes them.
+            //
+            // We want to enable local mode only for our own host app. All other
+            // addons should be loaded in global mode.
+            const hostAppWorkspaceDir = `${options.workspaceDir}/${app.name}`;
+            const isHostAppPath = resourcePath.includes(hostAppWorkspaceDir);
+
+            return isHostAppPath ? 'local' : 'global';
+          },
```

Comment:
I found a couple of cases where `mode` returns the incorrect value. Please see https://github.com/ijlee2/ember-container-query/pull/188 for the bug fix.